.travis.yml

```diff
@@ -6,15 +6,18 @@ notifications:
     on_failure: change
 
 php:
-  - 5.6
```

Comment:
The plugin may be being used outside of VIP, so a change in minimum PHP would be a breaking change, and therefore require a major version bump. For now, let's keep 5.6 as the minimum and have that as a separate discussion.

Reply:
rebased the PR to include this change from fix/travix